File: app/Produtos.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Produtos extends Model
{
    //

    use SoftDeletes;
    protected $fillable = [
        'nome' , 'modelo' , 'cor' , 'quantidade' , 'sku' , 'preco' , 'descricao' ,'image' , 'categoria_id' , 'dimensao', 'peso' , 'tag_name'
    ];

    public function categorias()
    {
        return $this->belongsTo(Categoria::class, 'categoria_id', 'id');
    }


    public function carrinhos(){
        return $this->belongsTo(Carrinho::class);
    }
}

File: decFusion_AppMobile/app/src/main/java/com/example/dacfusion/model/Produtos.kt
package com.example.dacfusion.model

import java.util.*

data class Produtos (
    
    var id:Int? = null ,
    var categoria_id:Int? = null,
    var nome:String? = null,
    var cor:String? = null,
    var quantidade:Int? = null,
    var preco:Double? = null,
    var modelo:String? = null,
    var descricao:String? = null,
    var sku:Int? = null ,
    var image:String? = null ,
    var dimensao:String? = null ,
    var peso:String? = null ,
    var tag_name:String? =  null ,
    var created_at:Date? = null,
    var updated_at:Date? = null

)



//    "id": 1,
//"categoria_id": 1,
//"nome": "teste",
//"cor": "azul",
//"quantidade": 10,
//"preco": 20,
//"modelo": "sdas",
//"descricao": "ds",
//"sku": 5678,
//"image": "produtos/8Yw6LAHYT7PSTNZwPZv4ceF1XaNGDaOqiLn1S9rU.jpeg",
//"created_at": "2020-11-19T01:11:45.000000Z",
//"updated_at": "2020-11-19T01:11:45.000000Z"